manage

```diff
@@ -304,7 +329,7 @@ pingLedger(){
   ledger_url=${1}
 
   # ping ledger web browser for genesis txns
-  local rtnCd=$(curl -s --write-out '%{http_code}' --output /dev/null ${ledger_url}/genesis)
+  local rtnCd=$(curl -s --write-out '%{http_code}' --output /dev/null ${ledger_url})
```

Comment:
Doesn’t this change the definition of “ledger_url”?  The point of ledger URL is that it uses the VON Network ledger browser convention of `/genesis` path returning the ledger file.

Reply:
Line 351 now adds the `/genesis` on on the call to pingUrl if it is a Ledger_URL, otherwise just passes the Genesis URL as is. 
I can change the variable name to just `ledger` instead of `ledger_url`?